### DISCUSSION TITLE
Unicode Character - 0xC290

### DISCUSSION THREAD

**Discussion by kothapraveen** [FreeTDS/freetds#544] [Q&A] [answered]
Hi @freddy77,

Currently we are using Unicode Character - **0xC290** to insert in DB.
But in logs am seeing ?????? symbol.

am seeing the below logs

error.c:517:SQLGetDiagRec(3, 0x1b5b680, 1, 0x7fff32d7a320, 0x7fff32d7a314, 0x7fff32d7a330, 512, 0x7fff32d7a318)
error.c:566:SQLGetDiagRec: "[FreeTDS][SQL Server]Error converting characters into server's character set. Some character(s) could not be converted"
error.c:517:SQLGetDiagRec(3, 0x1b5b680, 2, 0x7fff32d7a320, 0x7fff32d7a314, 0x7fff32d7a330, 512, 0x7fff32d7a318)
error.c:517:SQLGetDiagRec(3, 0x1b5b680, 1, 0x1b5cac0, 0x7fc3a88a28ec, 0x1b59e60, 511, 0x7fff32d78b0e)
error.c:566:SQLGetDiagRec: "[FreeTDS][SQL Server]Error converting characters into server's character set. Some character(s) could not be converted"

can you suggest which character we can use to support in FreeTDS?

**Comment by freddy77** [comment, marked as answer]
It could be possible the log is not 100% Unicode compatible. But from the errors you see you are trying to insert that character into some column that cannot store that character.
